app/templates/partials/messages-status-fr.html

```diff
@@ -34,32 +34,28 @@ <h3 class="heading-small">Livraison réussie</h3>
 
 <h3 class="heading-small">Échec</h3>
 <p>
-  Si un message n’arrive pas à être livré, vous pouvez réessayer après 72 heures ou bien essayer de joindre l’utilisateur par un autre moyen. Un message peut échouer pour plusieurs raisons :
+  Si un message n’arrive pas à être livré, vous pouvez réessayer après 72 heures ou bien essayer de joindre l’utilisateur par un autre moyen.
+</p>
+<p>
+  Un message peut échouer pour plusieurs raisons, nottament :
 </p>
 
-<ul class="list-bullet">
+<ul style="list-style-type: disc; margin-left: 2em; margin-bottom: 20px;">
```

Comment:
```suggestion
<ul class="list list-bullet">
```
This what we use on the "Terms" page

https://github.com/cds-snc/notification-admin/blob/1375836399263888f7b3a687312558aea60fc71c/app/templates/views/terms-of-use.html#L19